sdcm/cluster.py

```diff
@@ -2003,6 +2036,19 @@ def install_scylla(self, scylla_repo):
             self.remoter.run('sudo yum install -y python36-PyYAML', ignore_status=True)
             self.remoter.run('sudo yum install -y {}'.format(self.scylla_pkg()))
             self.remoter.run('sudo yum install -y scylla-gdb', ignore_status=True)
+        elif self.distro.is_sles15:
+            self.remoter.sudo('zypper install -y rsync tcpdump screen')
+            self.download_scylla_repo(scylla_repo)
+            # self.remoter.sudo('zypper mr -e Python_2_Module_x86_64:SLE-Module-Python2-15-SP3-Pool')
+            # self.remoter.sudo('zypper mr -e Python_2_Module_x86_64:SLE-Module-Python2-15-SP3-Updates')
+            # self.remoter.sudo('zypper mr -e Legacy_Module_x86_64:SLE-Module-Legacy15-SP3-Updates')
+            # self.remoter.sudo('zypper mr -e Legacy_Module_x86_64:SLE-Module-Legacy15-SP3-Pool')
+            self.remoter.sudo('SUSEConnect --product sle-module-legacy/15.3/x86_64')
+            self.remoter.sudo('SUSEConnect --product sle-module-python2/15.3/x86_64')
+            self.remoter.sudo('zypper install -y python2-PyYAML', ignore_status=True)
+            self.remoter.sudo('zypper install -y python3-PyYAML', ignore_status=True)
```

Comment:
Is this part of the official documentation?

Reply:
We don't have install guide for Suse now.